apps/mac/FluentUIReactNative-macOS/Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIconFile</key>
+	<string></string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
I'm missing where this is actually specified, but what's the domain we're using? Usually has some impact on device deployability

Reply:
By domain you mean? I haven't touched this file manually

Reply:
yeah we probably should though. Looks like the domain is in the build settings that are inlined in the pbxproj. We should choose it intentionally and not just accept the default

Reply:
Not sure what you mean by the domain, but I tried to make the swift-conversion iteration of this mirror the initial fabric-macos change you sent out last year. The Info.plist _should_ be the same as it was in that PR.